build.sbt

```diff
@@ -142,25 +142,11 @@ lazy val decline =
       mimaPreviousArtifacts := Set(organization.value %% moduleName.value % mimaPreviousVersion)
     )
     .jsSettings(
-      libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.3.0",
+      libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.4.0",
       coverageEnabled := false
     )
     .nativeSettings(
```

Comment:
This can also be written as `.platformsSettings(JSPlatform, NativePlatform)(...)`